### DISCUSSION TITLE
Lack of E in *stat wrappers

### DISCUSSION THREAD

**Discussion by Hypnotron** [ptitSeb/box64#295] [General]
I was wondering why the signatures for my_fstat, my_fstatat, and my_lstat are missing Es in wrapped/wrappedlibc_private.h (they were added in commit 80be548). There's a %noE comment, so I assumed the Es were intentionally omitted, but box64 hangs/segfaults on those calls due to arguments being in the wrong place unless I insert Es and recompile (change iFip to iFEip, etc.). What's happening?

**Comment by ptitSeb** [comment]
You're right, it's a mystake on my side. I'll fix that.
